R/RsquareAdj.R

```diff
@@ -35,11 +35,12 @@
 
 ## cca result: no RsquareAdj
 RsquareAdj.cca <-
-    function(x, ...)
+    function (x, nperm = 1000) 
```

Comment:
Function definition needs to have `...` as an argument because the generic has this argument.

For consistency, `nperm` should be renamed to `permutations` as per `anova.cca` etc.